### DISCUSSION TITLE
R-CMD-CHECK Github action failing, because MASS needs R 4.3?

### DISCUSSION THREAD

**Discussion by david6marsh** [community/community#26293] [Actions]
<p>I’ve been using Github actions successfully for a while to do R-CMD-Check on package himach. Now, on most platforms the checks fail because “MASS needs R &gt;= 4.3”. This is odd because (a) R 4.3 doesn’t exist AFAIK (b) MASS on CRAN needs R &gt;= 3.3.0.</p>
<p>CMD-CHECK works OK on my own machine, and on rhub and on CRAN and with check_win_release.</p>
<p>Is anyone else having this problem? Or has something changed in the config? The action is <a href="https://github.com/david6marsh/himach/blob/41b58b00d71af40daf7a579b51ddcd90c4eaa7ff/.github/workflows/check-standard.yaml" rel="noopener nofollow ugc">defined here in github</a>.</p>
<p>David</p>

**Comment by DominiqueMaucieri** [comment]
<p>I have been having this same issue and it only started last week.</p>

**Comment by david6marsh** [comment]
<p>Thanks. Good to know. In the meantime, using the old-fashioned methods for checks on other platforms worked, and himach is up on CRAN having passed its checks.</p>

**Comment by DominiqueMaucieri** [comment, marked as answer]
<p>I was able to fix my issue by updating my github actions R-CMD-check to v2, so maybe that will work for you, <a href="https://github.com/DominiqueMaucieri/quadcleanR/blob/ea096ca41be4c11f6c3b88955bbf2d8b2334571f/.github/workflows/R-CMD-check.yaml" rel="noopener nofollow ugc">which you can see here</a></p>

**Comment by david6marsh** [comment]
<p>Excellent. Works for me too. Thanks Dominique.</p>
